modules/ezcontent_paragraphs/modules/ezcontent_hero_media/src/Plugin/field_group/FieldGroupFormatter/HtmlElementHeroMediaDisplay.php

```diff
@@ -6,6 +6,7 @@
 use Drupal\Core\Form\FormState;
 use Drupal\Core\Template\Attribute;
 use Drupal\field_group\Plugin\field_group\FieldGroupFormatter\HtmlElement;
+use \Drupal\field_group\Element\HtmlElement as HtmlElementFormatter;
```

Comment:
We should not be using a \ at the start of the namespace.
```suggestion
use Drupal\field_group\Element\HtmlElement as HtmlElementFormatter;
```